emulator/app/Cargo.toml

```diff
@@ -25,8 +25,13 @@ gdbstub.workspace = true
 hex.workspace = true
 tock-registers.workspace = true
 zerocopy.workspace = true
+bitfield.workspace = true
+crc = "3.2.1"
```

Comment:
minor nit: this is the same package that is used in `emulator/bmc/pldm-fw-pkg/Cargo.toml`, so we might as well consolidate them into the main `Cargo.toml` so they don't accidentally get out of sync.

Reply:
Done